module4/preparation/Week2/IntroToAPIs.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title:  Pre-Work for Intro to APIs
+---
+
+1. Review [Query String Parameters](/module3/lessons/Week4/QueryStringParameters). Focus on reviewing *query string parameters* and *path parameters*. 

```

Comment:
I wonder if we should have some research question here to help them focus their review?  Or maybe in Mod4, this is a better release of responsibility...

Reply:
I'm leaning towards less structure here, I think the terms "Query string params" and "path params" are hopefully enough to focus their review.